src/System.Diagnostics.FileVersionInfo/src/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcore50-Windows_NT;
+      uap-Windows_NT;
```

Comment:
I just did the minimal set needed to build.  It's possible there is more cleanup to do.

You are correct that projects without packages don't need any "old" configurations.  Also for projects with packages we can remove old configurations and opt to harvest those assemblies from the previous package.